source.md

```diff
@@ -481,6 +481,7 @@ If you appreciate the content 📖, support projects visibility, give 👍| ⭐|
 - [TailorMade](https://github.com/jogboms/tailor_made) <!--stargazers:jogboms/tailor_made--> - Managing a Fashion designer's daily routine using a mixture of ReBLoC w/ Firebase Cloud Store & Functions by [Jeremiah Ogbomo](https://twitter.com/jogboms)
 - [Instory](https://github.com/InvertedX/instory) <!--stargazers:InvertedX/instory--> - Instgram story downloader with a beautiful UI [Sarath](https://twitter.com/_sarath_kumar)
 - [Spacex-Go](https://github.com/jesusrp98/spacex-go) [177⭐] - Simple yet powerful, open-source SpaceX launch tracker. [jesusrp98](https://twitter.com/jesusrp98).
+- [GitJournal](https://github.com/GitJournal/GitJournal) <!--stargazers:GitJournal/GitJournal--> - A Journaling App which stores your data in a Git Repo by [Vishesh Handa](https://github.com/vHanda)
```

Comment:
I think the app is great, but it lacks a little UX/UI to be perfect.

Reply:
Thanks. I agree with the UX part. Making the git integration was quite time consuming. I'm now mostly just going to focus on the UI.

Reply:
Ping. Do you think it would now qualify?